### PR TITLE
Add the document for failover overview

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
Signed-off-by: changzhen <changzhen5@huawei.com>

**What type of PR is this?**

/kind feature
/kind documentation

**What this PR does / why we need it**:

Add the document for failover overview.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @RainbowMango

**Comment by RainbowMango** [comment]
Please @Poor12 take a look.

File: sidebars.js [hunk 1]
```diff
@@ -69,6 +69,9 @@ module.exports = {
                     label: "Multi-cluster Failover",
                     items: [
                         "userguide/failover/failover",
```
Comment (Poor12) [inline sidebars.js:71]: Do we need to delete the original article?

**Review by Poor12 — COMMENTED** [review]
Picture should be placed in the /docs/resources.

File: sidebars.js (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply sidebars.js:71]: Yes, I plan to do it.

Comment (XiShanYongYe-Chang) [reply sidebars.js:71]: Updated, please take a look.

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**